exasol_integration_test_docker_environment/test/test_test_environment_nameservers.py

```diff
@@ -41,8 +43,8 @@ def test_no_nameserver(self):
         self.docker_environment_name = "test_no_nameserver"
         on_host_docker_environment, google_cloud_docker_environment = \
             self.test_environment.spawn_docker_test_environment(self.docker_environment_name)
-        self.assert_nameserver("")
-        close_environments(on_host_docker_environment, google_cloud_docker_environment)
+        with ContextSpawnedTestEnvironment(on_host_docker_environment, google_cloud_docker_environment):
```

Comment:
Usually, you try to generate the object for the context directly in the with statement

Reply:
spawn_docker_test_environment should return ContextSpawnedTestEnvironment and ContextSpawnedTestEnvironment should have to members for the on_host_docker_environment and google_cloud_docker_environment